src/query/allowed_objects.c

```diff
@@ -39,7 +41,9 @@ static const FunctionByName g_allowed_builtins[] = {
     (FunctionByName){.name = "dtoi2", .primary_arg = 0},
     (FunctionByName){.name = "dtoi4", .primary_arg = 0},
     (FunctionByName){.name = "dtoi8", .primary_arg = 0},
+    (FunctionByName){.name = "numeric_int2", .primary_arg = 0},
```

Comment:
I'm not sure why originally `numeric_int2/8` would be treated differently from their floating point counterparts. Since I'm treating them the same elsewhere I filled this in here as well.